tests/unit/test_transaction.py

```diff
@@ -840,11 +818,6 @@ def test_context_mgr_failure(self):
 
         self.assertEqual(api._committed, None)
 
-        session_id, txn_id, metadata = api._rolled_back
```

Comment:
It's not immediately obvious why we don't need to check `api._rolled_back` when `transaction.rolled_back` is `True`. Consider adding a comment.

Reply:
Done